Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -146,6 +159,11 @@ class XMLHttpRequestBase {
       } else {
         this.responseText += responseText;
       }
+      if(this.reponseType == '' || this.reponseType == 'text')
+          this.response = Object(this.responseText);
```

Comment:
Do we need `Object(...)` here? I know it's a `DOMString` for some unknown reason, but does it need to be `String` object here instead of just a primitive?